README.md

```diff
@@ -33,12 +33,12 @@ more details:
 - [Develop Adapter Extension](https://github.com/google-ai-edge/model-explorer/wiki/6.-Develop-Adapter-Extension)
 - [Limitations and Known Issues](https://github.com/google-ai-edge/model-explorer/wiki/7.-Limitations-and-Known-Issues)
 
-We invite you to participate in our future research studies on Model Explorer. Sign up [here](https://docs.google.com/forms/d/e/1FAIpQLScGOkQOIKmIzkt3P0ywhSfwbl-TRb2epEV5J8NTXEesZqc3vw/viewform).
+We invite you to participate in research studies on Model Explorer. Sign up [here](https://docs.google.com/forms/d/e/1FAIpQLScGOkQOIKmIzkt3P0ywhSfwbl-TRb2epEV5J8NTXEesZqc3vw/viewform).
 
-## Custom Adapters
+## Community-contributed adapters
```

Comment:
The heading '## Community-contributed adapters' is inconsistent with the previous heading '## Custom Adapters'. To maintain consistency, it should be reverted.

```diff
-## Community-contributed adapters
+## Custom Adapters
```